integration_tests/snapd/test_catkin_snap.py

```diff
@@ -21,12 +21,12 @@
 
 import integration_tests
 from snapcraft.tests import fixture_setup
-from snapcraft.internal.common import get_os_release_info
+from snapcraft.internal.os_release import OsRelease
 
 
 class CatkinTestCase(integration_tests.SnapdIntegrationTestCase):
 
-    @skipUnless(get_os_release_info().get('VERSION_CODENAME') == 'xenial',
+    @skipUnless(OsRelease().version_codename() == 'xenial',
```

Comment:
This will fail now with an error if there is no `VERSION_CODENAME` or `VERSION_ID`, won't it? The test should however be skipped if those optional fields are not present.
Maybe the method should just return `None` in that case and not raise an exception?

Reply:
I agree that the test should be skipped instead of toss an exception, but I don't think that's the responsibility of `OsRelease`, but rather something on the test base class. I'll add a helper method there.

Reply:
Done.